backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -165,6 +192,15 @@ describe('ContributionResolver', () => {
             }),
           )
         })
+
+        it('stores the create contribution event in the database', async () => {
+          await expect(EventProtocol.find()).resolves.toContainEqual(
+            expect.objectContaining({
+              type: EventProtocolType.CONTRIBUTION_CREATE,
+              userId: bibi.data.login.id,
```

Comment:
Please also check for contribution id and amount. You get the contribution id in the response of the test above

Reply:
done but take a look